src/main/kotlin/com/msg/gcms/global/webhook/config/DiscordWebhookConfig.kt

```diff
@@ -0,0 +1,14 @@
+package com.msg.gcms.global.webhook.config
+
+import okhttp3.OkHttpClient
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+
+@Configuration
+class DiscordWebhookConfig{
+
```

Comment:
```suggestion
@Configuration
class DiscordWebhookConfig {

```
GCMS 백엔드 컨벤션이 개판인거가 있긴한데, 보통 클래스명이랑 { 랑 띄워서 해서 띄우는게 나을거같아요